src/data/shortlist.h

```diff
@@ -200,8 +199,9 @@ class BinaryShortlistGenerator : public ShortlistGenerator {
   // import text shortlist from file
   void import(const std::string& filename, double threshold);
   // generate blob from vectors
-  std::vector<char> generateBlob() const;
-  void saveBlobToFile(std::vector<char> blob, const std::string& filename) const;
+  std::vector<char> blob;
```

Comment:
This is a member variable?  It makes sense if it's effectively an allocated memory store.  But then treat it like one: it gets an _ and an comment.

Reply:
OK. I will fix the name.